docs/pages/development/1-development.rst

```diff
@@ -64,12 +64,17 @@ All              ``make run-generators``
 Docs Generation
 ***************
 
-To generate the documentation, you'll need ``python3``, ``sphinx`` and a bunch of other stuff.
-After you have installed ``python3`` (and ``pip3`` if this not already installed) run
+To generate the documentation, you'll need to build a docker image from the docs directory:
 
 .. code-block:: shell
 
-    pip3 install -r docs/requirements.txt
+   docker build -t nutsfoundation/nuts-node-docs .
```

Comment:
perhaps add the `cd docs` command here, since me and my fellow copy pasters don't read the line above and combine all in 1 code block:

```
.. code-block:: shell

   cd docs
   docker build -t nutsfoundation/nuts-node-docs .
   docker run --rm -v $PWD:/docs nutsfoundation/nuts-node-docs make html
```